ui/src/VoxettaUserIcon.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+import {LitElement, html, css} from 'lit-element';
+
+import {Icon} from '@material/mwc-icon';
+import {IconButton} from '@material/mwc-icon-button';
+
+/**
+ * Button responsible for enabling the user to record and upload audio files. 
+ */
+export class VoxettaUserIcon extends LitElement {
+
+    render() {
+        // TO DO (ashley) : stylize and add an actual icon
```

Comment:
nit: // TODO(ashley):

Reply:
forgot to remove after I added the icon--now removed--but will remember this style in the future